### PR TITLE
Topic/ghc

### PR DESCRIPTION
**Pull request by tmcgilchrist**
! @jystic

### CODE REVIEW

**Comment by tmcgilchrist** [comment]
! @erikd-ambiata

**Comment by erikd-ambiata** [comment]
The checks seem to be missing some 8.0 builds. Egf optparse, eithert, show etc

**Comment by tmcgilchrist** [comment]
Yeah I'll add the other things, I just did this one since it had a 8.0.1 lock file.

File: framework/master.toml [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
 
 [build.branches-7-10]
   GHC_VERSION = "7.10.2"
-  CABAL_VERSION = "1.22.4.0"
+  CABAL_VERSION = "1.24.0.0"
   HADDOCK = "true"
   HADDOCK_S3 = "$AMBIATA_HADDOCK_BRANCHES"
+
+[build.branches-8-0]
+  GHC_VERSION="8.0.1"
```
[the remark this answers is not in the document]
Comment (erikd-ambiata) [reply framework/master.toml]: @etorreborre We're haven't upgraded to 8.2 yet.

**Comment by tmcgilchrist** [comment]
Or 8.0.2 unfortunately

**Pull request merged**